### PR TITLE
fix for umbracoUserLogin not pruned

### PR DESCRIPTION
**Pull request by matthew-hair**
### Prerequisites

Umbraco version 7.13.x.

### Description

Issue #4661. This change fixes the database queries that are supposed to prune the umbracoUserLogin table of rows that are stale by more than 15 days. The current code checks for equality with 15 days prior to the current date. I changed the code to check for less-than-or-equal-to 15 days prior instead.

To test, log into the back office, and then close the browser window without explicitly logging out, and do not log in with the same user account for at least 15 days. Then restart the application pool. The umbracoUserLogin table should be pruned so that no records with lastValidatedUtc > 15 days ago remain.

### CODE REVIEW

**Comment by emmaburstow** [comment]
Hey @matthew-hair 

Thanks for the pull request. We'll take a look at it and let you know if we need anything from you before we merge.

All the best,

Emma

File: src/Umbraco.Core/Persistence/Repositories/UserRepository.cs [hunk 1]
```diff
@@ -267,8 +267,8 @@ public int ClearLoginSessions(TimeSpan timespan)
 
             var fromDate = DateTime.UtcNow - timespan;
 
-            var count = Database.ExecuteScalar<int>("SELECT COUNT(*) FROM umbracoUserLogin WHERE lastValidatedUtc=@fromDate", new { fromDate = fromDate });
-            Database.Execute("DELETE FROM umbracoUserLogin WHERE lastValidatedUtc=@fromDate", new { fromDate = fromDate });
+            var count = Database.ExecuteScalar<int>("SELECT COUNT(*) FROM umbracoUserLogin WHERE lastValidatedUtc=<@fromDate", new { fromDate = fromDate });
```
Comment (nul800sebastiaan) [inline src/Umbraco.Core/Persistence/Repositories/UserRepository.cs]: Surely this should be `<=` and not `=<` ?

**Comment by matthew-hair** [comment]
> If you can make the logic valid for both queries we can pull this in! 👍

Thanks, I'm glad you caught my typo! I tried submitting an amendment to my original pull request, but I think I just made another commit (78822ae) instead. Git is new to me, and I'm struggling a bit with it.

**Comment by nul800sebastiaan** [comment]
Wonderful @matthew-hair - the additional commit just gets added to the PR so that's great!

Thanks very much for sorting this one out! 🎖

**Pull request merged**